src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.v2.js

```diff
@@ -13,8 +13,14 @@ export function transformFormToVAOSCCRequest(state) {
 
   if (provider?.identifier) {
     practitioners = [
-      data.communityCareProvider.identifier.find(item => item.system === 'PPMS')
-        ?.value,
+      {
+        id: {
```

Comment:
This matches the format the CC request practitioners when fetching appointments. It doesn't matter yet, because the HSRM implementation isn't reading this field.